scripts/scraper-ng/vfile-reporter-summary.js

```diff
@@ -31,6 +31,14 @@ function reporter(files) {
 function summarize(files) {
   const count = countRules(files);
 
+  if (Object.entries(count).length === 0) {
+    return [
+      "",
+      chalk.yellow(chalk.underline("Summary")),
+      "No issues found. Hooray! 🎉"
```

Comment:
Almost too cheerful.

Reply:
Don't worry, you probably won't see it often